test_rclcpp/test/test_n_nodes.py.in

```diff
@@ -0,0 +1,45 @@
+# generated from test_rclcpp/test/test_n_nodes.py.in
+
+import os
+
+from launch.legacy import LaunchDescriptor
+from launch.legacy.exit_handler import primary_exit_handler
+from launch.legacy.launcher import DefaultLauncher
+
+
+def test_@TEST_NUM_NODES@_nodes():
+    ld = LaunchDescriptor()
+
+    for i in range(0, @TEST_NUM_NODES@):
+        cmd = ['@TEST_EXECUTABLE1@']
+        env = None
+        if '@TEST_RMW_IMPLEMENTATION@':
+            env = dict(os.environ)
+            env['RMW_IMPLEMENTATION'] = '@TEST_RMW_IMPLEMENTATION@'
+        ld.add_process(
+            cmd=cmd,
+            name='node_with_name' + str(i),
```

Comment:
Maybe add an underscore between `node_with_name` and the number for readability?

Reply:
Changed in 5a9c83ed1c9fed1b567bcf8a431a0a849e384aca